pyproject.toml

```diff
@@ -12,9 +12,18 @@ maintainers = [
     {name = "Nicolas Tessore", email = "n.tessore@ucl.ac.uk"},
 ]
 classifiers = [
-    "Programming Language :: Python :: 3",
     "License :: OSI Approved :: MIT License",
     "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Scientific/Engineering :: Astronomy",
+    "Topic :: Scientific/Engineering :: Physics",
+    "Typing :: Typed",
```

Comment:
Should we add this classifier before adding a mypy check / `py.typed` file?

Reply:
Arguably not, will remove